cmd/block-generator/generator/server.go

```diff
@@ -75,12 +96,27 @@ func getBlockHandler(gen Generator) func(w http.ResponseWriter, r *http.Request)
 			return
 		}
 
-		gen.WriteBlock(w, round)
+		writeResponse(w, gen.WriteBlock(w, round))
+	}
+}
+
+func getAccountHandler(gen Generator) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// The generator doesn't actually care about the block...
+		account, err := parseAccount(r.URL.Path)
+		if err != nil {
+			fmt.Fprintf(w, err.Error())
```

Comment:
What does this do? Should we return an error using `http.Error()` like below?

Reply:
Yes, good idea.